tests/integration/nmstate_service_test.py

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2022 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+import os
+
+import yaml
+import pytest
+
+from .testlib.cmdlib import exec_cmd
+from .testlib.assertlib import assert_state_match
+
+TEST_YAML_CONTENT = """
+---
+interfaces:
+- name: eth1
+  state: up
+  mtu: 1500
+  ipv4:
+    address:
+    - ip: 192.0.2.252
+      prefix-length: 24
+    - ip: 192.0.2.251
+      prefix-length: 24
+    dhcp: false
+    enabled: true
+  ipv6:
+    address:
+      - ip: 2001:db8:2::1
+        prefix-length: 64
+      - ip: 2001:db8:1::1
+        prefix-length: 64
+    autoconf: false
+    dhcp: false
+    enabled: true
+"""
+
+CONFIG_DIR = "/etc/nmstate"
+TEST_CONFIG_FILE_NAME = "nmstate-test.yml"
```

Comment:
Can we test with a pair of files like `'01-nmstate-test.yml` and `02-nmstate-test.yml`, and also to be sure that they are being applied at proper order 01 can create a "dummy" and 02 can modify that dummy so if they are applied at correct order it will work.

Reply:
Sure. Let me do that.